docs/docs/ref-01-top-level-api.md

```diff
@@ -89,9 +89,13 @@ If the optional callback is provided, it will be executed after the component is
 
 > Note:
 >
-> `React.render()` replaces the contents of the container node you
-> pass in. In the future, it may be possible to insert a component to an
-> existing DOM node without overwriting the existing children.
+> `React.render()` controls the contents of the container node you pass in*. Any existing DOM elements
```

Comment:
Ohh, I see. I think it's better to leave out the footnote demarkation. The content reads well flowing into the next block without it.